test/connection/beginTransaction.js

```diff
@@ -104,36 +94,44 @@ describe('.beginTransaction([callback])...', () => {
       });
     });
     it('...should make transactional queries visible only to the connection that the transaction was started on until commit() is called (at transactional isolation level \'read committed\').', (done) => {
-      // set commitment level to 1 (Read committed)
-      odbc.connect(`${process.env.CONNECTION_STRING};CMT=1`, (error1, connection1) => {
+      let connectionString1Keywords = '';
+      let connectionString2Keywords = '';
+      if (global.dbms === 'ibmi')
+      {
+        // set commitment level to 1 (Read committed)
+        connectionString1Keywords= ';CMT=1';
+        // set commitment level to 1 (Read committed)
+        connectionString2Keywords = ';CMT=1;CONCURRENTACCESSRESOLUTION=1';
+      }
+      else if (global.dbms === 'mssql')
+      {
+        connectionString2Keywords = ';DMConnAttr=[1227]=32';
```

Comment:
FYI, this is a unixODBC-specific attribute

Reply:
Definitely something that I will have to tackle in the future... Not sure what else would be a good way of setting these driver-specific values without the driver header file (not that even that would help in these test cases). Maybe a generic "setConnectionAttribute" function where I pass in the magic numbers?

Reply:
Yeah, I think going this way is fine for now. Might be worth adding a Windows check that exits with an explanatory message, though.

A generic interface would be good. Can always add niceties on top of it if needed. I don't think it's necessary if all it's needed for at the moment is testing.